app/views/registrations/_payment_intents.html.erb

```diff
@@ -1,7 +1,7 @@
 <div>
   There are pending payments for this registration. Please wait until they are confirmed before paying again.
 
-  <% @registration.stripe_payment_intents.pending.each do |payment_intent| %>
+  <% @registration.payment_intents.pending.each do |payment_intent| %>
```

Comment:
Does the `pending` scope here cover enough possible statuses? (ie, created/partial/failed/canceled are all excluded from it, some of which should perhaps be included)

Reply:
No, I don't think so. This is only meant to show actual _pending_ intents. If an intent has failed, we will allow the user to restart it while loading the UI widget, instead of forcing them to stare at a list detailing that it failed.

Reply:
Ah interesting - in this case, is our current status mapping for `pending` too broad? (wca_status: [stripe_statuses])
```ruby
pending: %w[pending requires_capture processing requires_confirmation requires_action],
```